src/pages/workshop/nllp-2022.md

```diff
@@ -32,6 +32,428 @@ The workshop proceedings are available [here](https://aclanthology.org/events/em
+			Leonard Bongard (Technical University of Darmstadt); Lena Held (Technical University of Darmstadt); Ivan Habernal (Technische Universitt Darmstadt)</td>
+		</tr>				
+		<tr bgcolor="#FDF6F0">
+			<td style="padding-top:5px;padding-bottom:5px;padding-right:10px;padding-left:10px" nowrap="nowrap" valign="top">16:55 - 17:00</td>
+			<td style="padding-top:5px;padding-bottom:5px;padding-right:10px;padding-left:10px"><b><a href="" target="_blank">Legal and Political Stance Detection of SCOTUS Language</a></b> <br />
+			Noah Bergam (Columbia University); Emily Allaway (Columbia University); Kathleen McKeown (Columbia University and Amazon (Amazon Scholar))</td>
+		</tr>
+		<tr bgcolor="#FDF6F0">
+			<td style="padding-top:5px;padding-bottom:5px;padding-right:10px;padding-left:10px" nowrap="nowrap" valign="top">17:00 - 17:05</td>
+			<td style="padding-top:5px;padding-bottom:5px;padding-right:10px;padding-left:10px"><b><a href="" target="_blank">Can AMR Assist Legal and Logical Reasoning?</a></b> <br />
+			Nikolaus Schrack (University of Copenhagen); Ruixiang Cui (University of Copenhagen); Hugo A. Lopez (Technical University of Denmark); Daniel Hershcovich (University of Copenhagen)</td>
+		</tr>		
+		<tr bgcolor="#FDF6F0">
+			<td style="padding-top:5px;padding-bottom:5px;padding-right:10px;padding-left:10px" nowrap="nowrap" valign="top">17:05 - 17:10</td>
+			<td style="padding-top:5px;padding-bottom:5px;padding-right:10px;padding-left:10px"><b><a href="" target="_blank">LawngNLI: a long-premise benchmark for evaluating models on in-domain generalization to long contexts and implication-based retrieval</a></b> <br />
+			William Bruno; Dan Roth</td>
+		</tr>		
+		<tr bgcolor="#FDF6F0">
+			<td style="padding-top:5px;padding-bottom:5px;padding-right:10px;padding-left:10px" nowrap="nowrap" valign="top">17:10 - 17:15</td>
+			<td style="padding-top:5px;padding-bottom:5px;padding-right:10px;padding-left:10px"><b><a href="" target="_blank">Legal Prompting: Teaching a Language Model to Think Like a Lawyer</a></b> <br />
+			Fangyi Yu (Thomson Reuters); Lee Quartey (Thomson Reuters); Frank Schilder (Thomson Reuters)</td>
+		</tr>
+		<tr bgcolor="#FDF6F0">
+			<td style="padding-top:5px;padding-bottom:5px;padding-right:10px;padding-left:10px" nowrap="nowrap" valign="top">17:15 - 17:30</td>
+			<td style="padding-top:5px;padding-bottom:5px;padding-right:10px;padding-left:10px"><b>Joint Q&amp;A</b></td>
+		</tr>	
+		<tr bgcolor="#F8E2CF">
+			<td style="padding-top:5px;padding-bottom:5px;padding-right:10px;padding-left:10px" nowrap="nowrap" valign="top">17:30 - 18:30</td>
+			<td style="padding-top:5px;padding-bottom:5px;padding-right:10px;padding-left:10px"><b>Panel Discussion (From NLLP to legal NLP: The Future of the Field) &amp; Best Presentation Award</b></td>
+		</tr>		
+ </tbody>
+</table>
+
+---
```

Comment:
Committees header

This metadata section needs to be removed from the body and only kept at the top